isis/src/osirisrex/objs/OsirisRexOcamsCamera/assets/orex_ocams_addendum_v05.ti

```diff
@@ -0,0 +1,290 @@
+\begintext
```

Comment:
@kledmundson Apologies for getting to this late, can these addendum files be removed? They are already in the ISIS data area so I am not sure why they are being included here